$NetBSD: patch-configure,v 1.2 2015/08/26 11:31:02 leot Exp $

o Older freetype2 versions have a different path for ftheader.h, also check for
  it in order to avoid forcing dependency to the pkgsrc version of
  graphics/freetype2. For more information please give a look to pkg/49878.
o test(1) does not support the "==" operator. Use the "=" operator to compare if
  two strings are identical.

--- configure.orig	2014-12-05 14:51:58.000000000 +0000
+++ configure
@@ -14233,7 +14233,9 @@ done
 	  CPPFLAGS="$freetype_cflags $CPPFLAGS"
 	fi
 	ac_fn_cxx_check_header_mongrel "$LINENO" "config/ftheader.h" "ac_cv_header_config_ftheader_h" "$ac_includes_default"
+	ac_fn_cxx_check_header_mongrel "$LINENO" "freetype/config/ftheader.h" "ac_cv_header_freetype_config_ftheader_h" "$ac_includes_default"
 if test "x$ac_cv_header_config_ftheader_h" = xyes; then :
+elif test "x$ac_cv_header_freetype_config_ftheader_h" = xyes; then :
 
 else
   as_fn_error $? "\"ftheader.h not found\"" "$LINENO" 5
@@ -16194,7 +16196,7 @@ echo "==================================
 
 
 # Warning message about Xft support
-if test "x$TEST_XFT" == "x" ; then
+if test "x$TEST_XFT" = "x" ; then
 	echo ""
 	echo "==================================== Warning ================================================"
 	echo "Configure has detected that your FOX library was compiled without Xft support."
